stories/side-navigation/side-navigation.stories.js

```diff
@@ -1303,7 +1307,13 @@ export const nestedListPopover = () => `<div class="fddocs-container" style="mar
             </ul>
         </div>
         <div class="fd-popover__popper" aria-hidden="false" id="popoverA12">
-            <ul class="fd-nested-list fd-nested-list--popover fd-nested-list--compact">
+            <div class="fd-nested-list fd-nested-list--popover">
+                <div class="fd-nested-list__popover-header">
+                    Popover Header
+                </div>
```

Comment:
Please follow the markup from fund-ngx.

Reply:
Can you please give more info/describe more.
In last change, when i tried to follow the Ngx markup [here](https://github.com/SAP/fundamental-styles/pull/2692#pullrequestreview-746182875). there was concern that we can not use `fd-nested-list--text-only`.
I am not sure, what changes i need to make now.

Reply:
@InnaAtanasova 
I am not sure about what markup i need to take from Ngx. what i have i already done is taken from Ngx markup.
It looks fine to me.
for comparing purpose i had created https://github.com/SAP/fundamental-ngx/pull/6387 PR, so comparision will be better. which is already linked in this PR.

Also as there is no design available and i am not clarified on exactly what i need to make change.
I am closing this PR.